README.md

```diff
@@ -40,17 +42,16 @@ If you'd like to have a similar SPA wrapper for the OSS version of FingerprintJS
 
 ## Table of Contents
 
-- [Documentation](#documentation)
+- [Requirements](#requirements)
 - [Installation](#installation)
 - [Getting Started](#getting-started)
-- [Support + Feedback](#support--feedback)
+- [Support and Feedback](#support-and-feedback)
+- [Documentation](#documentation)
 - [License](#license)
 
-## Documentation
+## Requirements
 
-This library uses [Fingerprint Pro](https://fingerprint.com/github/) under the hood.
-- To learn more about Fingerprint Pro read our [product documentation](https://dev.fingerprint.com/docs).
-- To learn more about this SDK, there is a [Typedoc-generated SDK Reference](https://fingerprintjs.github.io/fingerprintjs-pro-spa) available.
+- For TypeScript users: Typescript 4.5 or higher
```

Comment:
@TheUnderScorer @ilfa Does it make sense to define some kind of [support policy](https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc?pvs=4#599e723a44f74dbab6f1ec796a572b19) here or not really?

Reply:
For TypeScript? I'm not sure we need to. @ilfa?

Reply:
It looks complicated =( https://github.com/microsoft/TypeScript/blob/main/SUPPORT.md

Reply:
I think that we can omit it in that case because it's not related to the runtime, just the types.